pkg/rancher-desktop/preload/extensions.ts

```diff
@@ -595,6 +642,71 @@ class Client implements v1.DockerDesktopClient {
         };
       });
     },
+    subscribeToEvents: (options: DockerEventSubscriptionOptions = {}, callback: DockerEventCallback): DockerEventSubscription => {
```

Comment:
Having a default for the first argument but _not_ the second seems odd. Since the callback is mandatory, make it the first argument instead?